purchases/src/main/kotlin/com/revenuecat/purchases/models/Transaction.kt

```diff
@@ -0,0 +1,20 @@
+package com.revenuecat.purchases.models
+
+import android.os.Parcelable
+import com.revenuecat.purchases.getDate
+import kotlinx.android.parcel.Parcelize
+import org.json.JSONObject
+import java.util.Date
+
+@Parcelize
+data class Transaction(
+    val transactionId: String,
+    val productId: String,
+    val purchaseDate: Date
+) : Parcelable {
+    constructor(productId: String, jsonObject: JSONObject) : this(
```

Comment:
Do you think we should change this secondary constructor to be the primary one? I prefer this way so the jsonObject is not stored. It is also easy to test since a Transaction class can easily be created by calling the primary constructor. Reason I am not convinced is that there're two constructors 🤷